circle.yml

```diff
@@ -67,5 +67,5 @@ dependencies:
 
 test:
   override:
-    - cargo test --features "dev" -- --nocapture
+    - cargo test --features "dev" -- --nocapture --verbose
```

Comment:
need verbose here?

Reply:
With verbose we will know the exact fail reason.